components/upload/interface.tsx

```diff
@@ -15,7 +15,7 @@ export interface RcFile extends File {
 
 export interface RcCustomRequestOptions {
   onProgress: (event: { percent: number }, file: RcFile) => void;
-  onError: (error: Error) => void;
+  onError: (error: Error, response?: any, file?: UploadFile) => void;
```

Comment:
the type of `file` should be `RcFile`

Reply:
Even if the [code define](https://github.com/ant-design/ant-design/blob/88ae028321e31b0270c9a48aec51abacbbdc5125/components/upload/Upload.tsx#L141) is `UploadFile`?

Reply:
ok, I agree